openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -843,4 +843,11 @@ a[href].link-text-grey,
   padding: 0;
   border: none;
   background-color: transparent;
+  cursor: pointer;
+  opacity: 0;
+  transition: 0.3s ease-in;
+  width: fit-content;
+  &:hover {
+    opacity: 100;
+  }
 }
```

Comment:
Can we move this styling to component specific less file? also the hover effect seems to be applied on the button itself, it should be on whole displayName component

Reply:
Removed these styles from app.less and fixed the hover effect, it is applied on the whole displayName component.